### PR TITLE
Top communities feature vasanth

### PR DESCRIPTION
**Pull request by vasanthakumar-a**
On the Home page, the right sidebar shows a list of the top communities according to the frequency of posts in a week (total posts count in the last 7 days)

### CODE REVIEW

**Review by mithun30052001 — APPROVED** [review]
Working fine without conflicts

File: app/controllers/communities_controller.rb [hunk 1]
```diff
@@ -52,8 +54,24 @@ def set_community
     @community = Community.find(params[:id])
   end
 
+  def find_all_communities
+    @communities = Community.all
```
Comment (iMagesh) [inline app/controllers/communities_controller.rb:58]: Don't use .all method

File: app/controllers/public_controller.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class PublicController < ApplicationController
 
   def index
-    @communities = Community.all.limit(5)
+    @communities = Community.all.order("post_count_this_week desc").limit(5)
```
Comment (iMagesh) [inline app/controllers/public_controller.rb:4]: .all is not required here

File: config/application.rb [hunk 3]
```diff
@@ -11,6 +11,8 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
 
+    config.time_zone = "Kolkata"
```
Comment (iMagesh) [inline config/application.rb:14]: shouldn't this be "Asia/Kolkata" ?

**Review by iMagesh — CHANGES_REQUESTED** [review]
Please check my comments, update it all and let me know.

File: app/controllers/public_controller.rb (hunk 2 above)
Comment (vasanthakumar-a) [reply app/controllers/public_controller.rb:4]: Resolved it

File: config/application.rb (hunk 3 above)
Comment (vasanthakumar-a) [reply config/application.rb:14]: Resolved it

File: app/controllers/communities_controller.rb (hunk 1 above)
Comment (vasanthakumar-a) [reply app/controllers/communities_controller.rb:58]: Communities will be displayed with an "infinite scroll feature" further

**Pull request merged**